lib/src/firebase_auth_mocks_base.dart

```diff
@@ -37,6 +37,14 @@ class MockFirebaseAuth implements FirebaseAuth {
     }
   }
 
+  set mockUser(MockUser user) {
+    _mockUser = user;
+    // Update _currentUser if already sign in
+    if (_currentUser != null) {
+      _currentUser = user;
+    }
+  }
```

Comment:
We need this setter because, in the integration test, we need to test a button for users to update their customClaim or other data.